wasm-transform/src/output.rs

```diff
@@ -94,7 +94,7 @@ impl<'a, A: Output> Output for &'a [A] {
 }
 
 /// This implem
-impl<'a, A: Output> Output for Vec<A> {
+impl<A: Output> Output for Vec<A> {
```

Comment:
Can we correct the comment now we're at it? "This implem" seem unfinished.

Reply:
Added.